src/faq/theme.css

```diff
@@ -61,22 +71,27 @@ li>faqanswer {
   color: #4A90E2;
   text-decoration: none;
 }
+
 #faqcontent {
   margin: 0 auto 60px auto;
   display: block;
   width: 94%;
   max-width: 675px;
 
 }
-h1 span.short-faq {
-  display: none;
-}
 
 h1 span.long-faq {
   color: #616161;
   display: block;
 }
+
 .faqa {
     text-decoration: none;
+    cursor: pointer;
     color: #2196F3;
 }
+
+.jp-QuestionMark {
```

Comment:
Since you're using a *global* CSS class `jp-QuestionMark`, which is meant to be used by anybody who wants to use that image, in this local context, you should give it a second class that is specific to your extension. Otherwise, anyone else who tries to use the question mark image after your CSS has been loaded will have its size hard coded. Even if you're pretty sure the image is just for your extension, since it's being offered up globally, may as well treat it like a common resource.